live-tests/README.adoc

```diff
@@ -51,11 +51,21 @@ To use this:
               $(which cargo-nextest) \
               root@YOUR_SCRIMLET_GZ_IP:/zone/oxz_switch/root/root
 
-3. On that system, unpack the tarball with:
+3. Copy omdb as well.
+
+     e.g., scp \
+              /home/dap/omicron-work/target/debug/omdb \
+              root@YOUR_SCRIMLET_GZ_IP:/zone/oxz_switch/root/root
+
```

Comment:
You don't need to do this.  It's already in the switch zone.

Reply:
oh wow, i never realized!